test/bats/tests/templates/use_http_send_template.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sdenynamehttpsend
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sDenyNameHttpSend
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            invalidName:
+              type: string
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8sdenynamehttpsend
+        violation[{"msg": msg}] {
+          input.review.object.metadata.name == input.parameters.invalidName
+          response := http.send({"method": "get", "url": "https://github.com/"})
```

Comment:
Ironically, any kind of "get" in our tests makes me nervous. It's a good thing this is disabled!